src/spreadsheets.js

```diff
@@ -407,7 +407,10 @@ function getRows(spreadsheetConfig, options, sheetPredicate, callback) {
                     });
                     var recordRows = [];
                     rows.slice(2).forEach(function(row) {
-                        recordRows.push(_.object(_.zip(headerRow, row)));
+                        var asObject = _.transform(_.zip(headerRow, row), function(result, pair) {
```

Comment:
I agree, the underscore version would be much preferred here, unfortunately `_.object` is not defined in lodash and I wasn't able to find a suitable replacement.